apiserver/pkg/model/converter_test.go

```diff
@@ -642,6 +670,16 @@ func tolerationToString(toleration *api.PodToleration) string {
 	return "Key: " + toleration.Key + " Operator: " + toleration.Operator + " Effect: " + toleration.Effect
 }
 
+func TestFromCrdToAPIJobs(t *testing.T) {
+	jobs := []*rayv1api.RayJob{&JobNewClusterTest}
+	job := FromCrdToAPIJobs(jobs)[0]
```

Comment:
Could you please help pre-allocate here?
https://github.com/ray-project/kuberay/blob/747708b7cc123eb270ada8f84dbc4a988836db03/apiserver/pkg/model/converter.go#L447